docker-compose.yml

```diff
@@ -40,7 +40,6 @@ services:
       - DATABASE_URL=postgres://postgres:cms@db/hitech_apd
       - NODE_ENV=development
       - DISABLE_SAME_SITE=yes
-      - DISABLE_SECURE_COOKIE=yes
```

Comment:
I'm assuming that I can remove this env var. Please correct me if I'm wrong!

Reply:
I think that's right